flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -255,6 +256,19 @@ class AndroidFlutterLocalNotificationsPlugin
   Future<void> deleteNotificationChannel(String channelId) =>
       _channel.invokeMethod('deleteNotificationChannel', channelId);
 
+  Future<List<ActiveNotification>> getActiveNotifications() async {
+    final List<Map<dynamic, dynamic>> activeNotifications =
```

Comment:
Sorry missed this last time but does this work if `dynamic` was replaced with `Object`?

Reply:
Trying to cast to another type throws an Exception at runtime so this signature should be kept for now